.coveragerc

```diff
@@ -2,14 +2,19 @@
 omit =
     urllib3/packages/*
     urllib3/contrib/*
+    test/*
+    dummyserver/*
+    */site-packages/*
+    /usr/bin/*
 
 [report]
+show_missing = true
 exclude_lines =
     except ImportError:
     pass
     import
     raise
     .* # Platform-specific.*
```

Comment:
This can also be replaced with more specific exclusions, and/or remove this comment from areas that are covered by the tox envlist matrix.  especially the 'AppEngine' code removed from coverage.